arcgis-ios-sdk-samples/Display information/Query features with Arcade expression/QueryFeaturesArcadeExpressionViewController.swift

```diff
@@ -0,0 +1,112 @@
+// Copyright 2021 Esri
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import UIKit
+import ArcGIS
+
+class QueryFeaturesArcadeExpressionViewController: UIViewController {
+    @IBOutlet var mapView: AGSMapView! {
+        didSet {
+            mapView.map = makeMap()
+            // Set the touch delegate.
+            mapView.touchDelegate = self
+        }
+    }
+    
+    /// The arcade expression evaluation operation.
```

Comment:
```suggestion
    /// The Arcade expression evaluation operation.
```
According to the [terminology](https://csreview.esri.com/terminology-guide) guide it should be capitalized.